src/openseadragon.js

```diff
@@ -2607,7 +2607,8 @@ function OpenSeadragon( options ){
          *      jpg:  true,
          *      png:  true,
          *      tif:  false,
-         *      wdp:  false
+         *      wdp:  false,
+         *      webp: true
```

Comment:
Just a couple lines down from here there's an example that shows setting webp as supported; now that it is supported by default maybe we should change that example to something less likely to be supported by default, like bmp.

Reply:
OK, I've pushed a commit for this change.